github/create-release.yml

```diff
@@ -33,13 +33,34 @@ steps:
       Write-Host "GitHub release is pre-release because of $Env:TAG"
       Write-Host "##vso[task.setvariable variable=isPreRelease]true"
       Write-Host "##vso[task.setvariable variable=compareWith]lastRelease"
+      Write-Host "##vso[task.setvariable variable=releaseNotes]$Env:RELEASE_NOTES"
+      Write-Host "##vso[task.setvariable variable=isDraft]false"
```

Comment:
Why are we setting it to false? 🤔

Reply:
I would always use draft releases - For both full & preview releases

Reply:
Although you could argue preview is not required, fair point.

Reply:
Yeah, it was bc we only have added the titles to the full releases and so the current setup of pre-releases is still 'complete' in the sense that only having `Install-Package -Name ...` is enough as release notes.